apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/OperationExtensions.kt

```diff
@@ -36,16 +35,22 @@ import kotlin.jvm.JvmOverloads
  * @param indent the indentation string to be repeated for each level of indentation in the encoded document. Must be a string
  * containing only whitespace. If [indent] is an empty String the encoded document will be compact. Otherwise the encoded
  * document will be more human-readable.
- * @param customScalarAdapters configured instance of custom GraphQL scalar type adapters. Default adapters are used if this
+ * @param responseAdapterCache configured instance of custom GraphQL scalar type adapters. Default adapters are used if this
```

Comment:
nit: `configured instance of GraphQL operation response adapters cache. Default adapters are used if this`?